package.json

```diff
@@ -135,8 +135,6 @@
   },
   "dependencies": {
     "fast-json-stable-stringify": "^2.0.0",
-    "graphql-tag": "^2.10.1",
-    "prop-types": "^15.6.0",
```

Comment:
Glad we got rid of `prop-types` too!

Reply:
Yep, I've noticed we weren't using it and we probably just forgot to remove it